front/public/cgi-bin/printenv.sh

```diff
@@ -0,0 +1,9 @@
+404: Script Not Found
+===================
+
+The script you're looking for was not found OR YOU WERE TRYING TO HACK MY SITE.
+
+NICE TRY. I HAVE LOGGED YOUR IP ADDRESS AND YOU WILL BE HACKED!!1
+
+
+Please report vulnerabilities at [the issue tracker](https://github.com/dikaiosune/rust-dashboard/issues).
```

Comment:
I thought it would be fun to show this to people who try to exploit heartbleed. Do you have a good name for this file?